### PR TITLE
man: add missing --remove option from aur sync

### PR DESCRIPTION
**Pull request by columbarius**
I wouldn't add a new repo-add section to the man page like in the aur-build man page, since aur-sync forwards it to aur-build, but it's an repo-add option, so I also could add a new section in the man page.

### CODE REVIEW

**Comment by rafasc** [comment]
In this case, the reference should be `(aur build -R)` because `aur sync` doesn't talk to `repo-add` directly, it just passes the `-R` option to `aur build`, which happens to pass it it to `repo-add`.

Otherwise looks good to me.

nitpick: "Add remove from repo option" is a bit hard to read, I would reword the subject, e.g.
`man: add missing --remove option from aur sync`

**Review by AladW — APPROVED** [review]

**Comment by AladW** [comment]
Looks good to me. Note you can easily merge/reword the commits with `git rebase`, e.g. `git rebase -i HEAD~2`.

**Comment by columbarius** [comment]
Branch squashed and rebased onto master

**Comment by columbarius** [comment]
Thanks for merging!
